src/index.html

```diff
@@ -7,12 +7,66 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <!-- Font -->
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <!--  -->
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="Desctop-img"
+        class="card__img"
+      />
+
+      <div class="card__content">
+        <div class="card__info">
+          <span class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </span>
+          <span class="card__description">Product code: 195434</span>
+        </div>
+        <div class="card__rating">
+          <div class="card__rating-stars">
+            <div class="card__rating-star"></div>
+            <div class="card__rating-star"></div>
+            <div class="card__rating-star"></div>
+            <div class="card__rating-star"></div>
+            <div class="card__rating-star"></div>
+          </div>
+          <div class="card__rating-text">Reviews: 5</div>
```

Comment:
Use the semantic tag for text
```suggestion
          <p class="card__rating-text">Reviews: 5</p>
```